src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(x) {
+    return function(y) {
+      count += 1;
+
+      if (count < 3 || count % 2 !== 0) {
+        return x + y;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
let's invert this condition — try to find «errors» first, then process everything else

Reply:
done!